SalesforceFoundation/gem
# Adding Apex class access needed for SGE to our Profile and Perm Set

- This change is only for our testing purposes, so I don't think it requires any change text.

# Critical Changes

# Changes

# Issues Closed

# New Metadata

# Deleted Metadata

# Testing Notes

- Run the dev_org or qa_org flow.
- Login as Gary Gift Officer.
- Ensure that the Single Gift Entry form can be filled out and saves correctly.
- Also check that "read" access to NPSP Relationships and Affiliations has been removed (we've already tested that auto-soft credits work without this access)
